resources/configurationFiles/analysisRNAseqGRCh38.xml

```diff
@@ -37,17 +34,20 @@
         <cvalue name='GENOME_STAR_INDEX_50' value='${hg38IndexDirectory}/STAR/STAR_2.7.6a_Gencode31/STAR_2.7.6a_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_50bp' type='string'/>
         <cvalue name='GENOME_STAR_INDEX_100' value='${hg38IndexDirectory}/STAR/STAR_2.7.6a_Gencode31/STAR_2.7.6a_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_100bp' type='string'/>
         <cvalue name='GENOME_STAR_INDEX_200' value='${hg38IndexDirectory}/STAR/STAR_2.7.6a_Gencode31/STAR_2.7.6a_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_200bp' type='string'/>
-        <cvalue name='GENOME_KALLISTO_INDEX' value='${hg38IndexDirectory}/kallisto/kallisto-0.46.0_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_k31/kallisto-0.46.0_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31.noGenes.index' type='string'/>
+        <cvalue name='GENOME_KALLISTO_INDEX'
+                value='${hg38IndexDirectory}/kallisto/kallisto-0.46.0_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_k31/kallisto-0.46.0_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31.noGenes.index'
+                type='string'/>
         <cvalue name='ARRIBA_KNOWN_FUSIONS' value='${hg38BaseDirectory}/tools_data/arriba/known_fusions_hg38_GRCh38_v${ARRIBA_VERSION}.tsv.gz' type='string'/>
         <cvalue name='ARRIBA_TAGS' value='${ARRIBA_KNOWN_FUSIONS}' type='string'/>
         <cvalue name='ARRIBA_BLACKLIST' value='${hg38BaseDirectory}/tools_data/arriba/blacklist_hg38_GRCh38_v${ARRIBA_VERSION}.tsv.gz' type='string'/>
         <cvalue name='ARRIBA_PROTEIN_DOMAINS' value='${hg38BaseDirectory}/tools_data/arriba/protein_domains_hg38_GRCh38_v${ARRIBA_VERSION}.gff3' type='string'/>
         <cvalue name='ARRIBA_CYTOBANDS' value='${hg38BaseDirectory}/tools_data/arriba/cytobands_hg38_GRCh38_v${ARRIBA_VERSION}.tsv' type="string"/>
 
         <!-- Hg38 Finger printing-->
-        <cvalue name="fingerprintingSitesFile_hs38" value="${hg38DatabaseDirectory}/fingerprinting/hovestadt_v1.1/snp138Common.n1000_hg38.vh20140318.bed" type="path"/>
-        <cvalue name="fingerprintingSitesFile" value="${fingerprintingSitesFile_hs38}" type="path"
+        <cvalue name="fingerprintingSitesFile_hg38"
+                value="${hg38DatabaseDirectory}/fingerprinting/hovestadt_v1.1/snp138Common.n1000_hg38.vh20140318.bed"
```

Comment:
Should that not be `/omics/odcf/reference_data/legacy/ngs_share/assemblies/hg_GRCh38/databases/fingerprinting/methylCtools-fingerprint/`? That's where the data is installed in the moment. We could also add as symlink there.